lib/real.js

```diff
@@ -89,9 +90,68 @@ SecureWorker.getQuoteData = function getQuoteData(quote) {
 
 // Do a remote attestation of a given quote. Quote can come from this or some other machine.
 // Returns a signed remote attestation statement on successful remote attestation.
-// Inputs and outputs are an arraybuffer.
-SecureWorker.getRemoteAttestation = function getRemoteAttestation(quote) {
-  throw new Error("Not implemented.");
+// Input is an arraybuffer containing the quote, and a callback function structured at function(result,err)
+// The RA signature is returned using the callback function as an arraybuffer
+// If an error occurs, result will be undefined and err will contain a string
+SecureWorker.getRemoteAttestation = function getRemoteAttestation(quote,callbackFn) {
+  var self=this;
+  var quote_data = self.getQuoteData(quote);
+
+  var quote_str = String.fromCharCode.appply(null,new UInt16Array(quote_data));
+
+  const ra_url = "https://api.trustedservices.intel.com/sgx/attestation/v3/report";
+  var payload = {"isvEnclaveQuote":quote_str};
+
+  //populate Ocp-Apim-Subscription-Key with the subscription key from the API portal
+  var options = {
+    method: "POST",
+    headers: {
+      "Ocp-Apim-Subscription-Key": ""
+    }
+  };
```

Comment:
maybe populate the Ocp-Apim-Subscription-Key from a getRemoteAttestation() parameter

Reply:
Added in next commit